### PR TITLE
Tidy imports

### PR DESCRIPTION
**Pull request by MarinPostma**
This pr is an offspring of my work on policies. It simply make imports a bit tidier.

### CODE REVIEW

**Review by penberg — APPROVED** [review]

**Comment by honzasp** [comment]
Does it mean that we should now put imports from `crate` and `self` _after_ other imports? I thought that our convention was to put `use crate`-s first, but I actually prefer to put them last (more precisely, 1. external crates, 2. `crate`, 3. `super`, 4. `self`).

**Comment by MarinPostma** [comment]
@honzasp my reference here would be burntsushi, and it goes like this:
1. std imports
2. external imports
3. crate imports (I don't distinguish between self, super and crate tbh, sorting kinda takes care of that itself)

for ref: 
- https://github.com/rust-lang/regex/blob/master/src/compile.rs
- https://github.com/BurntSushi/ripgrep/blob/master/crates/cli/src/wtr.rs

File: server/src/datastore/crud.rs [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 use futures::{Future, StreamExt};
 use guard::guard;
 use serde_derive::{Deserialize, Serialize};
 use serde_json::json;
 
+use super::engine::TransactionStatic;
+use super::query::{Mutation, QueryOp, QueryPlan, RequestContext, SortBy, SortKey};
+use super::value::EntityMap;
+use super::QueryEngine;
+use crate::datastore::expr::{BinaryExpr, BinaryOp, Expr, PropertyAccess, Value as ExprValue};
```
[the remark this answers is not in the document]
Comment (MarinPostma) [reply server/src/datastore/crud.rs:14]: that's a good question, likely because they were auto-imported

[the remark this answers is not in the document]
Comment (MarinPostma) [reply server/src/datastore/crud.rs:14]: or coding style? I rarely use super myself, and usually prefer absolute paths

[the remark this answers is not in the document]
Comment (dekimir) [reply server/src/datastore/crud.rs:14]: But in this PR, you changed 3 out of 4 `use crate::datastore` into `use super`.  And I'm asking: why not the 4th?

[the remark this answers is not in the document]
Comment (MarinPostma) [reply server/src/datastore/crud.rs:14]: probably auto-import, no specific reason, this is carved out of a much larger pr, and things have moved a lot, and I hadn't noticed. I'll change them back

**Pull request merged**